.github/workflows/dependabot_pr.yml

```diff
@@ -0,0 +1,37 @@
+name: Dependabot PR actions
+on: pull_request
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: GitHub App token
+        id: github_app_token
+        uses: tibdex/github-app-token@v1.5.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+          installation_id: 22958780
+
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ steps.github_app_token.outputs.token }}
+
+      - name: Update the changelog
+        uses: dangoslen/dependabot-changelog-helper@v1
```

Comment:
By default, it runs for if the PR has a `dependabot` label. I updated that within the `dependabot` config here - https://github.com/opensearch-project/OpenSearch/blob/main/.github/dependabot.yml#L7-L9
You will have to update the `dependabot.yml` label config as well.

Reply:
Updated. Thanks for letting me know!